openmdao/core/problem.py

```diff
@@ -172,11 +175,10 @@ def __init__(self, model=None, driver=None, comm=None, name=None, **options):
 
         self._initial_condition_cache = {}
 
-        # Status of the setup of _model.
-        # 0 -- Newly initialized problem or newly added model.
-        # 1 -- The `setup` method has been called, but vectors not initialized.
-        # 2 -- The `final_setup` has been run, everything ready to run.
-        self._setup_status = 0
+        self.PRE_SETUP = 0
```

Comment:
I would not make these constants attributes of Problem because it'll cause circular imports if you try to use them in system.py for example.  Maybe we need a constants.py file that contains nothing but constants and then anybody could include that without any issues.